patterns/post-header/post-header-style-1-default.php

```diff
@@ -21,7 +21,7 @@
 	</div>
 	<!-- /wp:group -->
 
-	<!-- wp:post-excerpt {"fontSize":"large"} /-->
+	<!-- wp:newspack-block-theme/subtitle {} /-->
```

Comment:
When editing a template we get placeholder texts for dynamic blocks like "Title", etc. It would be nice to have the same for the Article Subtitle block, which currently renders nothing in that context:

<img width="1463" alt="image" src="https://github.com/user-attachments/assets/67a2cecf-cb79-4506-a527-a235fee50f95">

Reply:
Good catch! Fixed in 2baa2251d8966ee6ccd17924eaedd9e13c2ecf71